tools/RAiDER/delay.py

```diff
@@ -123,7 +126,7 @@ def tropo_delay(args):
             return None, None
 
         try:
-            weather_model.write2HDF5(weather_model_file)
+            weather_model.write2NETCDF4(weather_model_file)
```

Comment:
I think it would be good to leave the HDF5 method in there, and change to a generic "write" method that takes a file type as a parameter in order to maintain backwards compatibility with anything else in the codebase.